### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by AbdelrhmanAmin**
Added logic helper functions modules:
 - Operate.js
 - Calculate.js

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @AbdelrhmanAmin  👋

**Good project so far, pretty easy to understand the logic, and good use of object destructuring. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    return n1 - n2;
+  } if (operation === '+') {
+    return n1 + n2;
+  } if (operation === '/' || operation === '÷') {
+    return n1 / n2;
+  } if (operation === '*') {
+    return n1 * n2;
+  } if (operation === '%') {
+    return n1 % n2;
+  }
```
Comment (mikenath223) [inline src/logic/operate.js]: - [x] Kindly use Big.js operations instead, replace mathematical operations with Big.js operations(plus, minus, etc).

Comment (mikenath223) [inline src/logic/operate.js]: - [x] Also note that this-%- is not a modulus operator but a percentage operator. Please make the change and divide by 100 or multiply by 0.01.

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @AbdelrhmanAmin :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**